dependencies.gradle

```diff
@@ -69,5 +77,8 @@ ext.packages = [
                 "core": "com.infinum.jsonapix:core:${versions.jsonapix}",
                 "processor": "com.infinum.jsonapix:processor:${versions.jsonapix}",
                 "retrofit": "com.infinum.jsonapix:retrofit:${versions.jsonapix}",
+        ],
+        "junit"      : [
+                "core": "junit:junit:${versions.junit}"
```

Comment:
Nothing blocking here, just wondering if we can use more up-to-date version of JUnit 
`org.junit.jupiter:junit-jupiter-*:5.8.2`
Asking because this version seems to not be maintained anymore. 
If yes, I can share the dependencies I have on JDE. If not, no problem, keep using this

Reply:
You're right, we can bump this up 👍